libs/block-editor/src/lib/extensions/components/suggestions/suggestions.component.ts

```diff
@@ -196,6 +197,21 @@ export class SuggestionsComponent implements OnInit, AfterViewInit {
         this.initContentletSelection();
     }
 
+    /**
+     * Set items visible based on filter
+     *
+     * @param {string} filter
+     * @memberof SuggestionsComponent
+     */
+    filterItems(filter: string = '') {
+        this.items.forEach((item) => {
+            item.visible = filter
+                ? item.label.toLowerCase().includes(filter.trim().toLowerCase())
+                : true;
+        });
```

Comment:
Why this `visible` instead of create a new array with the results? also, why not create a new array? that's the best practice, never mutate he original.